src/mark.js

```diff
@@ -280,7 +280,19 @@ class Mark { // eslint-disable-line no-unused-vars
         default:
             return `()(${str})`;
         case "complementary":
-            return `()([^\\s${lsJoin}]*${str}[^\\s${lsJoin}]*)`;
+            lsJoin = lsJoin.length ?
+                `^\\s${lsJoin}` :
+                [
+                    "A-Za-z\-", // Basic Latin + dash
+                    "\u00AD", // Soft hyphen
+                    "\u00AF", // Macron
+                    "\u00B4", // Acute accent
+                    "\u00B8", // Cedilla
+                    "\u00C0-\uD7FF",
+                    "\uF900-\uFDCF",
+                    "\uFDF0-\uFFEF"
```

Comment:
Wouldn't it be helpful here to double escape the unicode characters here? This is the result in the console when you enable the `debug` option:

![2017-02-04_150024](https://cloud.githubusercontent.com/assets/8310677/22618999/f4a49e1e-eaea-11e6-8e8a-ddaca5b85471.png)

This looks pretty strange to me.